### ISSUE TITLE
Change default staticfiles configuration to avoid .py files being accessible publicly

### ISSUE THREAD

**Issue by EmilStenstrom** [EmilStenstrom/django-components#260]
Today we suggest the following configuration:

> Modify STATICFILES_DIRS (or add it if you don't have it) so django can find your static JS and CSS files:
> 
> ```
> STATICFILES_DIRS = [
>     ...,
>     BASE_DIR / "components",
> ]
> ```

This means we will get automatic bundling of static files via the collectstatic management command. But since all .py files are stored in the same components directory, we will bundle the .py files too. This could be a security issue, if sensitive information is included in the .py file.

The suggested fix is to ignore all the .py files, by subclassing and replacing the staticfiles app, so that it ignored .py files.

**Comment by lemontheme** [comment]
On it!

**Comment by lemontheme** [comment]
PR above has been merged and released in v0.27. Going forward, users can choose what level of risk is acceptable to them.

If you're looking for an easy fix to stop .py and .html files being collected by collectstatic, modify your INSTALLED_APPS as follows:

```
INSTALLED_APPS = [
    ...,
    #  "django.contrib.staticfiles",  # <-- remove
    "django_components",
    "django_components.safer_staticfiles",  # <-- add
    ...,
]
```

More details are provided in the new Security Notes section in the README.

**Comment by VojtechPetru** [comment]
:1st_place_medal: Thanks for being so quick with the fix guys! :clap:
